<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(171, 171, 171);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Apple Garamond Light</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(223, 223, 223)</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_0">
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>-410</x>
       <y>350</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(89, 89, 89)</string>
     </property>
     <property name="text">
      <string>24-05-2017</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>14:05:02</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="equipo">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>250</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sucursal">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>370</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QComboBox" name="tipo">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>480</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Boletera</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cajero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Salida</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Publicidad</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="nomPlaza_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>240</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>Equipo</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_10">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>Sucursal</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_11">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>470</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>Tipo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bconfirmar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>680</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-COLOR:rgb(23, 150, 208);
COLOR:rgb(255, 255, 255);
BORDER-RADIUS:80%</string>
     </property>
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bsalir2">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>680</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-COLOR:rgb(23, 150, 208);
COLOR:rgb(255, 255, 255);
BORDER-RADIUS:80%</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_12">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>80</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>Configuracion del equipo</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_13">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../iconos/logo2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="bvolver">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>660</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-COLOR:rgb(222, 25, 25);
COLOR:rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>VOLVER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bsalir">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>660</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-COLOR:rgb(222, 25, 25);
COLOR:rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>SALIR</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>971</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(59, 59, 59)</string>
     </property>
     <property name="text">
      <string>Es necesario reiniciar el equipo para aplicar los cambios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bvolver_2">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>350</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-COLOR:rgb(23, 150, 208);
COLOR:rgb(255, 255, 255);
BORDER-RADIUS:80%</string>
     </property>
     <property name="text">
      <string>Reiniciar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="avisoInserta_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>---&gt; Inserta tu boleto &lt;---</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mayan</family>
       <pointsize>45</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(118, 232, 69)</string>
     </property>
     <property name="text">
      <string>Bienvenido!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bboletoPerdido_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>781</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rrrrgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:20%</string>
     </property>
     <property name="text">
      <string>Boleto Perdido</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../cajeroFisico/.designer/iconos/warning.png</normaloff>../../cajeroFisico/.designer/iconos/warning.png</iconset>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0.00568182, x2:1, y2:1, stop:1 rgba(255, 255, 255, 0))</string>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bnCerrarSesion">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>610</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20%;
background:rgb(233, 63, 63);
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>CERRAR SESIÓN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>310</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagenes/outputQR.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>80</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>14:05:02</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>24-05-2017</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0.00568182, x2:1, y2:1, stop:1 rgba(255, 255, 255, 0))</string>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>630</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagenes/9570924_300x300.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>2310</x>
       <y>730</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagenes/l.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="nomPlaza_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Apple Garamond Light</family>
       <pointsize>48</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(101, 179, 0)</string>
     </property>
     <property name="text">
      <string>ESTACIONAMIENTO</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="nomLoc_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans Georgian</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:rgb(158, 158, 158)</string>
     </property>
     <property name="text">
      <string>CDMX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>630</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagenes/l.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bnConfirmarDesc">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>380</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(66, 174, 70);
color:rgb(255, 255, 255);
border-radius:20%</string>
     </property>
     <property name="text">
      <string>CONFIRMAR DESCUENTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bnCancelarDesc">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>460</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(233, 63, 63);
color:rgb(255, 255, 255);
border-radius:20%;</string>
     </property>
     <property name="text">
      <string>CANCELAR DESCUENTO</string>
     </property>
    </widget>
    <widget class="QLabel" name="mensajeServer">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>351</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Droid Serif</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bnApagar_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>680</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(104, 2, 2);
color:rgb(255, 255, 255);
border-radius:20%</string>
     </property>
     <property name="text">
      <string>Apagar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bnborrar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>810</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(104, 2, 2);
color:rgb(0, 0, 0);
border-radius:20%</string>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bconfirmavol">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>860</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rrgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:20%</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
